packages/neon/neon/pubspec.yaml

```diff
@@ -66,7 +67,7 @@ dev_dependencies:
     git:
       url: https://github.com/nextcloud/neon
       path: packages/neon_lints
-  test: ^1.24.9
+  test: ^1.24.3
```

Comment:
Was this intended?

Reply:
Yes, the dynamic_color package uses flutter_test in lib and has it as a normal dependency. This breaks some constraints and I had to downgrade this. The same is true for neon_dashboard because flutter_test from sdk and the latest test release are not compatible.